src/app/public/modules/ag-grid/ag-grid.service.ts

```diff
@@ -205,4 +210,12 @@ export class SkyAgGridService {
   private getIconTemplate(iconName: string): string {
     return `<i class="fa fa-${iconName}"></i>`;
   }
+
+  private onKeyPress(keypress: CellKeyPressEvent) {
+    const event: KeyboardEvent = keypress.event as KeyboardEvent;
+
+    if (event.key === 'Enter') {
+      keypress.api.startEditingCell({rowIndex: keypress.rowIndex, colKey: keypress.colDef.colId });
```

Comment:
You're missing a space between the opening curly brace of the object literal and the first property name, which is inconsistent with the space between the last property value and the closing curly brace. For clarity, I'd break the statement into multiple lines:

```
keypress.api.startEditingCell({
  rowIndex: keypress.rowIndex,
  colKey: keypress.colDef.colId 
});
```